src/styles/index.scss

```diff
@@ -8,9 +8,10 @@ $theme-font-path: '../../node_modules/uswds/src/fonts';
 $theme-hero-image: '../images/hero.png';
 
 //Custom themes
-@import './_custom-theme.scss';
+@import './custom-theme.scss';
 
 @import 'uswds';
 
 //Class overrides and custom classes
-@import './_class-overrides.scss';
+@import './class-overrides.scss';
+@import './documentation-page'
```

Comment:
I see you don't use `.scss` here. Do you need to? If not, stick to a convention in this file of including it or not at the end of imports.

Reply:
Good point - seems like it's not needed